src/test-utils/utils.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { act as reactDomAct } from 'react-dom/test-utils';
+
+export const act = ('act' in React ? React.act : reactDomAct) as typeof reactDomAct;
```

Comment:
can we import this from test-utils package?
I tried to do that and got a failed build, but I didn't invest much time in it

Reply:
test-utils package does not expose `act`, and I would not even do that, because it's supposed to be a temporary solution

Reply:
I meant to export it from test-utils https://github.com/cloudscape-design/test-utils/blob/main/src/core/dom.ts#L8

I don't think it's a temporary solution, we'll continue to support react 16 and 18 for the upcoming future so this condition will not change.

Reply:
It now exposes it for this very reason, no? https://github.com/cloudscape-design/test-utils/pull/58

Reply:
It will be exported from `utils` instead: https://github.com/cloudscape-design/test-utils/pull/62